lib/smartapi-connect.js

```diff
@@ -475,6 +475,25 @@ var SmartApi = function (params) {
 		return get_request('get_all_holding');
 	};
 
+    /**
+   	 * Description
+     * @method indOrderDetails
+     * @param {string} qParam
+     */
+    self.indOrderDetails = function (qParam) {
+        return get_request("ind_order_details",undefined, undefined, undefined, qParam);
```

Comment:
get_request("ind_order_details",undefined, undefined, undefined, qParam);

This is not coding practice to send too many null/unknown variables in method. make separate method for sake of clarity.

Reply:
Made separate method.